src/agent/main.cpp

```diff
@@ -192,17 +197,26 @@ static int realmain(int argc, char *argv[])
     }
 
     otbrLogInit(kSyslogIdent, logLevel, verbose);
-    otbrLogInfo("Running %s", OTBR_PACKAGE_VERSION);
-    otbrLogInfo("Thread version: %s", otbr::Ncp::ControllerOpenThread::GetThreadVersion());
-    otbrLogInfo("Thread interface: %s", interfaceName);
-    otbrLogInfo("Backbone interface: %s", backboneInterfaceName);
 
     for (int i = optind; i < argc; i++)
     {
-        otbrLogInfo("Radio URL: %s", argv[i]);
         radioUrls.push_back(argv[i]);
     }
 
+    {
```

Comment:
I don't see the motivation to combine logs into one line.

Reply:
Feeling it's more concise merging to one log line. 

I don't strongly insist on it, though. 

Will revert back to multiple lines.